examples/todos/src/index.js

```diff
@@ -5,7 +5,10 @@ import { Provider } from 'react-redux'
 import App from './components/App'
 import reducer from './reducers'
 
-const store = createStore(reducer)
+const store = createStore(
+  reducer,
+   window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
+)
```

Comment:
Maybe a comment stating it's not mandatory? I'm afraid it will confuse newcomers in the current state.